cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -350,7 +351,8 @@ private void createConfigTable(String configTableName) throws IOException {
     htd.addFamily(hcd);
     hcd.setMaxVersions(1);
 
-    tableUtil.createTableIfNotExists(getHBaseAdmin(), tableName, htd, null,
+    // the TableId below is ignored, since the HTableDescriptor is created without it.
```

Comment:
Right, this is because in this PR, I have not added queue tables to HBase namespaces yet, since there were other PRs that were working on this part of code to be ready for HBase namespaces. So right now, the ``HBaseTableDescriptor`` is created from the table name as a byte array. It does not have the notion of namespace in it and will be created in the default namespace. Eventually, the ``new HTableDescriptor(tableName)`` above will make way for ``HBaseTableUtil.getHTableDescriptor(TableId)``. I had to create the ``tableId`` right now because I removed the ``createIfNotExists`` methods that accepted String/byte array table names.